2-WebApp-graph-user/2-5-HybridFlow/README.md

```diff
@@ -0,0 +1,772 @@
+
+  <details>
+   <summary>Expand this section if you want to use this automation:</summary>
+
+    > **WARNING**: If you have never used **Azure AD Powershell** before, we recommend you go through the [App Creation Scripts guide](./AppCreationScripts/AppCreationScripts.md) once to ensure that your environment is prepared correctly for this step.
+  
+    1. On Windows, run PowerShell as **Administrator** and navigate to the root of the cloned directory
+    1. In PowerShell run:
+
+       ```PowerShell
+       Set-ExecutionPolicy -ExecutionPolicy RemoteSigned -Scope Process -Force
+       ```
+
+    1. Run the script to create your Azure AD application and configure the code of the sample application accordingly.
+    1. For interactive process - in PowerShell run:
+
+       ```PowerShell
+       cd .\AppCreationScripts\
+       .\Configure.ps1 -TenantId "[Optional] - your tenant id" -Environment "[Optional] - Azure environment, defaults to 'Global'"
+       ```
+
+       > Other ways of running the scripts are described in [App Creation Scripts guide](./AppCreationScripts/AppCreationScripts.md)
+       > The scripts also provide a guide to automated application registration, configuration and removal which can help in your CI/CD scenarios.
+
+  </details>
+
+### Manual Steps
+
+ > Note: You should skip the following steps if you've used the automation provided above.
+
+Follow the steps below for manually register and configure your apps
+
+<details>
+   <summary>Expand this section to manually register your app(s):</summary>
+  1. Sign in to the [Azure portal](https://portal.azure.com).
+  1. If your account is present in more than one Azure AD tenant, select your profile at the top right corner in the menu on top of the page, and then **switch directory** to change your portal session to the desired Azure AD tenant.
+
+#### Register the service app
+
+  1. Navigate to the [Azure portal](https://portal.azure.com) and select the **Azure Active Directory** service.
+  1. Select the **App Registrations** blade on the left, then select **New registration**.
+  1. In the **Register an application page** that appears, enter your application's registration information:
+     * In the **Name** section, enter a meaningful application name that will be displayed to users of the app, for example `HybridFlow-aspnetcore`.
+  1. Under **Supported account types**, select **Accounts in this organizational directory only**
+  1. Click **Register** to create the application.
+  1. In the app's registration screen, find and note the **Application (client) ID**. You'll need to use this value in your app's configuration files.
+  1. In the app's registration screen, select **Authentication** in the menu.
+     * If you don't have a platform added, select **Add a platform** and select the **Web** option.
+     * In the **Redirect URIs** section, enter the following redirect URIs.
+        * `https://localhost:44321/signin/`
```

Comment:
```suggestion
        * `https://localhost:44321/signin-oidc/`
```

Also, in the config file it looks like we configure a front-channel logout (/signout-oidc) uri but it doesnt seem to be in the manual instructions

Reply:
That is vestigial.

I'll get rid of it.